Mage.Sets/src/mage/cards/n/NornsDecree.java

```diff
@@ -0,0 +1,45 @@
+package mage.cards.n;
+
+import java.util.UUID;
+
+import mage.abilities.common.CombatDamageDealtToYouTriggeredAbility;
+import mage.abilities.common.PlayerAttacksTriggeredAbility;
+import mage.abilities.condition.common.AttackedPlayersPoisonedCondition;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.abilities.effects.common.DrawCardTargetEffect;
+import mage.abilities.effects.common.counter.AddPoisonCounterTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+
+/**
+ *
+ * @author alexander-novo
+ */
+public final class NornsDecree extends CardImpl {
+
+    public NornsDecree(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.ENCHANTMENT }, "{2}{W}");
+
+        // Whenever one or more creatures an opponent controls deal combat damage to
```

Comment:
Rules comment in the cards must be one line (if it's tooooooo long, you can divide it into sentences). It's important for searching. Comments auto-format in VSC can be disabled by settings ([example](https://stackoverflow.com/a/69472284/1276632)).